### PR TITLE
Andoird CD

### PR DESCRIPTION
**Pull request by faisal00813**
Generates Android artefacts

### CODE REVIEW

**Comment by vimmerru** [comment]
I am not sure that it is a good idea to have CI/CD without tests execution.

File: Jenkinsfile.cd [hunk 1]
```diff
@@ -558,6 +582,46 @@ def debPublishing(testEnv, directory, packageName, version, inStashName, outStas
...
+}
+
+
+def androidPublishArtifacts(buildEnv, packageVersion,arch) {
+    buildEnv.inside {
+        sh 'chmod -R 777 ci'
+
+        withCredentials([file(credentialsId: 'SovrinRepoSSHKey', variable: 'repo_key')]) {
+//            Creating directory using SFTP is not working. Hence creating directory using ssh and files are uploaded using SFTP in ci/android-zip-and-upload.sh
+            sh "ssh -v -oStrictHostKeyChecking=no -i '$repo_key' repo@192.168.11.115 mkdir -p /var/repository/repos/android/libindy/$env.BRANCH_NAME/$packageVersion-$env.BUILD_NUMBER"
```
Comment (jovfer) [inline Jenkinsfile.cd]: I suggest to move these lines into ci/android-zip-and-upload.sh script.

**Comment by jovfer** [comment]
As we haven't full CI (build + test) for Android yet, it's better to have at least check build as minimal CI.

**Comment by faisal00813** [comment]
@jovfer @vimmerru I have added test in CI script where a build for android architectures happen and serves as a test that the android build is is not broken.

**Comment by faisal00813** [comment]
Closing the PR in favour of #959

**Pull request closed without merging**